curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md

```diff
@@ -13,8 +13,9 @@ In ES5, we usually define a constructor function, and use the <code>new</code> k
 The class syntax simply replaces the constructor function creation:
 <blockquote>class SpaceShuttle {<br>&nbsp;&nbsp;constructor(targetPlanet){<br>&nbsp;&nbsp;&nbsp;&nbsp;this.targetPlanet = targetPlanet;<br>&nbsp;&nbsp;}<br>}<br>const zeus = new SpaceShuttle('Jupiter');</blockquote>
 Notice that the <code>class</code> keyword declares a new function, and a constructor was added, which would be invoked when <code>new</code> is called - to create a new object.<br>
-<strong>Note</strong><br>
-UpperCamelCase should be used by convention for ES6 class names, as in <code>SpaceShuttle</code> used above.
+<strong>Notes</strong><br>
+1. UpperCamelCase should be used by convention for ES6 class names, as in <code>SpaceShuttle</code> used above.
+2. The constructor method is a special method for creating and initializing an object created with a class. Learn more about it in the Object Oriented Programming section of the Javascript Algorithms And Data Structures Certification
```

Comment:
```suggestion
2. The constructor method is a special method for creating and initializing an object created with a class. You will learn more about it in the Object Oriented Programming section of the Javascript Algorithms And Data Structures Certification
```
This is a better wording since if anyone is going through the curriculum, they won't have to hop on to another challenge from in between somewhere else